### PR TITLE
Carve out textfix-formatter to monorepo package

### LINKED ISSUE #347: Carve out textfix-formatter to monorepo module
Currently, [textfix-formatter.ts](https://github.com/textlint/textlint/blob/cacbd643f61dbe9afb4cc78d577129b6984b653e/packages/textlint/src/fixer/textfix-formatter.ts "textfix-formatter.ts") is in `textlint` module.
We can carve out [textfix-formatter.ts](https://github.com/textlint/textlint/blob/cacbd643f61dbe9afb4cc78d577129b6984b653e/packages/textlint/src/fixer/textfix-formatter.ts "textfix-formatter.ts") and `formatters/` to a isolated module like `textlint-fixer-formatter`.

## Related

Import https://github.com/textlint/textlint-formatter to this monorepo #270

### PR DESCRIPTION
**Pull request by Leko**
This PR closes #347. And it related #270.

### CODE REVIEW

File: packages/textlint-fixer-formatter/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "textlint-fixer-formatter",
+  "version": "1.0.0",
+  "description": "TBD",
```
Comment (azu) [inline packages/textlint-fixer-formatter/package.json]: > textlint output formatter for fixer

Comment (Leko) [reply packages/textlint-fixer-formatter/package.json]: @azu Thank you for your suggestion.
I'll update description like your comment.

File: packages/textlint-fixer-formatter/package.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    "@types/node": "^8.0.57",
+    "cross-env": "^5.1.1",
+    "mocha": "^4.0.1",
+    "rimraf": "^2.6.2",
+    "ts-node": "^4.0.1",
+    "typescript": "^2.6.2"
+  },
+  "dependencies": {
+    "@textlint/kernel": "^1.0.3",
+    "chalk": "^2.3.0",
```
Comment (azu) [inline packages/textlint-fixer-formatter/package.json]: Previously, textlint use chalk 1.x.x.
Chalk 2.x.x remove `stripColor` method.

> Removed chalk.stripColor(). Use the strip-ansi package directly instead. 04cae22
> https://github.com/chalk/chalk/releases/tag/v2.0.0

File: packages/textlint/package.json [hunk 3]
```diff
@@ -59,6 +59,7 @@
     "textlint-rule-no-todo": "^2.0.0",
     "textlint-rule-preset-ja-spacing": "^2.0.0",
     "textlint-rule-preset-jtf-style": "^2.3.0",
+    "textlint-fixer-formatter": "^1.0.0",
```
Comment (azu) [inline packages/textlint/package.json]: Yay.

We can remove textlint-fixer-formatter's dependencies form `textlint`'s `dependencies` field`.
For example, we can remove chalk from textlint-self

File: packages/textlint-fixer-formatter/package.json (hunk 2 above)
Comment (Leko) [reply packages/textlint-fixer-formatter/package.json]: Oops. I downgraded it version to 1.1.3.

File: packages/textlint/package.json (hunk 3 above)
Comment (Leko) [reply packages/textlint/package.json]: I removed `chalk`, `diff`, `string-width` and `text-table` from textlint !

**Comment by azu** [comment]
- [ ] Add [Core Table](https://github.com/textlint/textlint/#core)
- [ ] Add simple README

```markdown
# textlint-fixer-formatter

> textlint output formatter for fixer

## Installation

    npm install textlint-fixer-formatter

## Usage

- [ ] TBD

## Tests

    npm test

## License

MIT
```

**Comment by Leko** [comment]
Thanks! I fixed two problems.

**Pull request merged**